### PR TITLE
Persian providers

### PR DESCRIPTION
**Pull request by zoli**
I've added persian local providers, Including Person and Internet.

### CODE REVIEW

File: src/Faker/Provider/fa_IR/Internet.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    protected static $safeEmailTld = array(
+        'com', 'ir', 'me', 'net', 'org',
+    );
+
+    protected static $tld = array(
+        'biz', 'com', 'info', 'ir', 'net', 'org',
+    );
+
+    protected static $lastNameAscii = array(
+        'ahmadi', 'rezaei', 'mohammady', 'salari', 'zare', 'chamran', 'hosseini', 'hasan', 'shahriari', 'karimi', 'majidi', 'moosavi', 'soleimani', 'ebrahimi', 'abbasi', 'majedi', 'nili', 'yazdi', 'jalili', 'babaei', 'ghasemi',
```
Comment (fzaninotto) [inline src/Faker/Provider/fa_IR/Internet.php]: We'll need more random data than that...

File: src/Faker/Provider/fa_IR/Person.php [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+        'هاله', 'هانا', 'هاني', 'هانيه', 'هایده', 'هدیه', 'هستی', 'هلاله', 'هما', 'همای', 'همراز', 'هنگامه', 'هوردخت', 'هورشید', 'هیلا', 'هیلا', 'هیلدا', 'هیوا',
+        'وارسته', 'واژه', 'واله', 'وانوشه', 'وجستا', 'وجیهه', 'ورتا', 'ورد', 'وردا', 'ورسا', 'ورنا', 'وستا', 'وشتا', 'وشتی', 'وصال', 'وندا', 'ونیژه', 'ویدا', 'ویرا', 'ویره', 'ویژه', 'ویس', 'ویستا', 'ویشکا', 'وینا',
+        'یارا', 'یاس', 'یاسمن', 'یاسمین', 'یاقوت', 'یزدانه', 'یکامه', 'یکتا', 'یگانه', 'یلدا', 'یوتاب',
+    );
+
+    protected static $lastName = array(
```
Comment (fzaninotto) [inline src/Faker/Provider/fa_IR/Person.php]: The list seems pretty limited. Also, could you point to the source?

File: src/Faker/Provider/fa_IR/Internet.php (hunk 1 above)
Comment (zidoms) [inline src/Faker/Provider/fa_IR/Internet.php]: Like how much?

Comment (fzaninotto) [inline src/Faker/Provider/fa_IR/Internet.php]: A few dozen names at least. Look at the other providers for an example.

One question: can't translitteration from arabic work?

File: src/Faker/Provider/fa_IR/Person.php (hunk 2 above)
Comment (zidoms) [inline src/Faker/Provider/fa_IR/Person.php]: I didn't found any specific source for last names, just added the names that i know.

**Comment by zidoms** [comment]
The text generating is the same as `ar_JO` just the regular expression has changed to match persian characters.

**Comment by fzaninotto** [comment]
OK for the text, sorry I thought the first method contained the text.
Also OK for the lastName, let's hope that someone can add more names in the future.

**Comment by zidoms** [comment]
Added more Ascii first and last names will add more persian last names in a second to.

**Comment by zidoms** [comment]
Ok, I think all issues mentioned above are resolved now.

**Comment by zidoms** [comment]
Is there anything else I should do to get this merged?

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**